### PR TITLE
Awesome book

### PR DESCRIPTION
**Pull request by LionRouge1**
Awesome books: with ES6

### CODE REVIEW

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @LionRouge1, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+});
+
+// Remove book function
+
+function removeBk() {
+  const Book = new Books();
+  Book.removeBook(this.id);
+  this.parentNode.parentNode.remove();
+  snaker('remove', '<b> Remove:</b> Book Removed successfully ');
+}
```
Comment (FrederickMih) [inline index.js:49]: - [ ] Hi @LionRouge1 , remember the main goal of this project is to understand the usage of ES6 arrow functions syntax and modules. Please let's stay away from traditional function expression syntax and make use of [ES6 arrow function syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions).

File: index.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+const btn = document.querySelectorAll('.btn');
+btn.forEach((element) => {
+  element.addEventListener('click', removeBk);
+});
+
+// navbar section
+const links = document.querySelectorAll('.links');
+links.forEach((element) => {
+  element.addEventListener('click', function () {
```
Comment (FrederickMih) [inline index.js:59]: - [ ] As pointed above, let's completely stay away from the `function` keyword.

File: modules/display.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+export default function displayBook(book) {
```
Comment (FrederickMih) [inline modules/display.js:1]: - [ ] I will be happy if we can start making use of [ES6 arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)

File: modules/flashMessage.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+const snakbar = document.getElementById('message-container');
+export default function snaker(type, mes, time = 4000) {
```
Comment (FrederickMih) [inline modules/flashMessage.js:2]: - [ ] Please no function keyword anymore. :angry:

File: modules/navbar.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
+export default function showBlock(e) {
```
Comment (FrederickMih) [inline modules/navbar.js:1]: - [ ] Please let's stay away from the function keyword in this project.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @LionRouge1 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @LionRouge1, 

Excellent work so far implementing the changes requested by the previous reviewer/s! 👌
You have implemented almost all the required features. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
- [ ] I can see that you have refactored some of the functions to use arrow syntax instead of the functions keyword but not all of them. Kindly, make use of the text editor's search functionality   and search for the keyword `function` (I recommend using VSCode), you will find two functions in the codebase that are still using it(previously highlighted by the reviewers) and one comment, please update them.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_N/A_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**